package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mozjpeg",
-  "version": "5.0.0",
+  "version": "5.0.1-rc0",
```

Comment:
Don't bump the version. I'll do that when I release a new version.

Reply:
i changed it because i want to test the fallback build script (otherwise, it will use the old prebuilt binary). the new ci build seems fine. it's all is good, i will change it back to `5.0.0` before merging?

Reply:
i changed it just to test the fallback build script. otherwise it's going to use `const url = `https://raw.githubusercontent.com/imagemin/mozjpeg-bin/v${pkg.version}/vendor/`;`. i'm changing it back now.